src/test/suite/editors/playgroundSelectedCodeActionProvider.test.ts

```diff
@@ -492,7 +492,7 @@ suite('Playground Selected CodeAction Provider Test Suite', function () {
     const testCodeActionProvider = new PlaygroundSelectedCodeActionProvider();
 
     beforeEach(async () => {
-      const fileName = path.join(os.tmpdir(), `regular-file-${uuidv4()}.js`);
+      const fileName = path.join(os.homedir(), `regular-file-${uuidv4()}.js`);
```

Comment:
Do we actually write to these locations in the tests?

Reply:
No, same as for the playground controller we discussed:
[playgroundController.ts](https://github.com/mongodb-js/vscode/pull/482/files/414c3a706ef7fabdeca601b9d77e5e2ed1e2a924#diff-48bf2c040e1a42186fc2a9be26388ca4cef5409d0c9bbcacee65052c5c765f13)

But seeing your concerns I revert back to tmpdir in tests :)

Reply:
Well if we’re not writing to it, it might be best to ensure that and use a path that isn’t actually writable (e.g. `/nonexistent` or similar)?